executor/src/main/java/org/k8loud/executor/cnapp/sockshop/SockShopProperties.java

```diff
@@ -0,0 +1,13 @@
+package org.k8loud.executor.cnapp.sockshop;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@Data
+@Component
+@ConfigurationProperties(prefix = "sockshop")
+public class SockShopProperties {
+    private String registerUserUrlSupplement = "register";
+    private String customersUrlSupplement = "customers";
```

Comment:
why no static?

Reply:
None of our Properties classes have static fields

Reply:
but those variables could be just static fields in a class IMO no need for Config bean

Reply:
Resolved in a private conversation